### ISSUE TITLE
`Cannot find module` errors

### ISSUE THREAD

**Issue by ehmicky** [netlify/zip-it-and-ship-it#110]
The following error is happening while bundling Functions:

```
In file "/opt/build/repo/functions/buttondown.js": Cannot find module 'isomorphic-unfetch/package.json' from '/opt/build/repo/functions' 
    node_modules/resolve/lib/async.js:121:35 
    node_modules/resolve/lib/async.js:274:39 processDirs
    node_modules/resolve/lib/async.js:281:32 isdir
    node_modules/resolve/lib/async.js:23:69 
    fs.js:167:21 FSReqCallback.oncomplete
```

**Comment by ehmicky** [comment]
Fixed by #112

**Comment by ehmicky** [comment]
Note: this error happens when a function file is requiring a file that does not exist. The error message is clear enough.
